clients/java-logger/shell/src/test/java/com/abcxyz/lumberjack/loggingshell/LoggingControllerTest.java

```diff
@@ -44,15 +44,20 @@ public class LoggingControllerTest {
   private static final String TEST_TRACE_ID = "testTraceId";
   private static final String TEST_EMAIL = "testEmail";
 
+  private static final String TEST_JVS_TOKEN =
+      "eyJhbGciOiJFUzI1NiIsImtpZCI6ImludGVnLWtleSIsInR5cCI6IkpXVCJ9.eyJhdWQiOlsiaHR0cCJdLCJleHAiOjE2NzcyODAzMjEsImlhdCI6MTY3NzI3NjcyMSwiaXNzIjoibHVtYmVyamFjay10ZXN0LXJ1bm5lciIsImp0aSI6IjkwMTdiZGJkLTViNDktNGVmOS1hZjllLTdmMDkxOWZmNGFiZiIsImp1c3RzIjpbeyJjYXRlZ29yeSI6InRlc3QiLCJ2YWx1ZSI6InRlc3QifV0sIm5iZiI6MTY3NzI3NjcyMSwic3ViIjoibHVtYmVyamFjay1pbnRlZyJ9.GzwrHJg2DbQAA64lOJcNGcBynFftIzY5eGUR9brHd9t6GTRImDHQUiuE4sQBYkDaKoEoPUh_cDca5UkGTbqmcg";
+
   private static final Timestamp TEST_TIMESTAMP =
       Timestamp.newBuilder().setSeconds(1577836800).build();
   private static final MockHttpServletRequestBuilder GET_REQUEST_BUILDER = get(REQUEST_PATH);
   private static final MockHttpServletRequestBuilder GET_REQUEST_BUILDER_WITH_EMAIL =
       get(REQUEST_PATH).requestAttr(TokenInterceptor.INTERCEPTOR_USER_EMAIL_KEY, TEST_EMAIL);
-  private static final MockHttpServletRequestBuilder GET_REQUEST_BUILDER_WITH_EMAIL_AND_TRACE_ID =
```

Comment:
What happen if justification is missing in the header. should we only require token when justification is enabled?

Reply:
This could be a blocker for future local testing (if we plan to add) when we want to test disabled case?

Reply:
Yes. If we want to test disabled case, we need to change it. But currently, I don't see any plans to test disabled case for now. In addition, we even removed requireJustification setting in GRPC e2e.